examples/typescript/audit.ts

```diff
@@ -22,11 +22,8 @@ let auditClient: audit.AuditClient;
 
 export async function getSubscriptionRegions(tenancyId: string) {
   // To retrieve the list of all available regions.
-  const listRegionSubscriptionsRequest: identity.requests.ListRegionSubscriptionsRequest = {
```

Comment:
@davidkhala I believe we can keep listRegionSubscriptionsRequest instead of passing in an object with { tenancyid }. listRegionSubscriptionsRequest is more descriptive in how the request might look like. The request object can take more than just tenancyId so its best to leave this code to show the type and other optional parameters the users may want to pass.